### PR TITLE
Feature/153 filter dashboard

### PR DESCRIPTION
**Pull request by todd-dembrey**
Adds basic filters for the dashboard. 

Based on django-select2, but expanded to use the multi-checkboxes from https://github.com/wasikuss/select2-multi-checkboxes (example: https://jsfiddle.net/wasikuss/7ak9skbb/)

Depending on merger order I will add other filters (see #95 #97)

### CODE REVIEW

File: opentech/apply/dashboard/tables.py [hunk 1]
```diff
@@ -13,3 +18,28 @@ class Meta:
...
+class Select2ModelMultipleChoiceFilter(filters.ModelMultipleChoiceFilter):
+    def __init__(self, *args, **kwargs):
+        label = kwargs.get('label')
+        kwargs.setdefault('widget', Select2MultiCheckboxesWidget(attrs={'data-placeholder': label}))
+        super().__init__(*args, **kwargs)
+
+
+class SubmissionFilter(filters.FilterSet):
+    round = Select2ModelMultipleChoiceFilter(queryset=get_used_rounds, label="Rounds")
+    page = Select2ModelMultipleChoiceFilter(queryset=get_used_funds, label='Funds')
```
Comment (zerolab) [inline opentech/apply/dashboard/tables.py]: I see the reasoning behind using `page`, but `fund` makes a lot more sense especially when it comes to query parameters

File: opentech/apply/dashboard/widgets.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    def build_attrs(self, *args, **kwargs):
+        attrs = super().build_attrs(*args, **kwargs)
+        attrs['class'] = attrs['class'].replace('django-select2', 'django-select2-checkboxes')
+        return attrs
+
+    @property
+    def media(self):
+        media = super().media
+        media.add_js([static('js/select2.multi-checkboxes.js'), static('js/django_select2-checkboxes.js')])
```
Comment (zerolab) [inline opentech/apply/dashboard/widgets.py]: Would be good to use the minified versions

File: opentech/apply/dashboard/tables.py (hunk 1 above)
Comment (zerolab) [reply opentech/apply/dashboard/tables.py]: missing lead filter

File: opentech/apply/dashboard/widgets.py (hunk 2 above)
Comment (todd-dembrey) [reply opentech/apply/dashboard/widgets.py]: using https://jscompress.com/ i get the followin stats, so dont really feel a desperate need to minify.

`Stats: 47.9% compression, saving 1.22 kb`

**Pull request merged**